pkgs/development/compilers/llvm/common/default.nix

```
@@ -109,12 +105,23 @@ let
           "${patchDir}/${p}";
       };
   };
-
-  tools = lib.makeExtensible (
-    tools:
+in
+makeScopeWithSplicing' {
+  inherit otherSplices;
+  f =
+    llvmPackages:
```

```suggestion
  f =
    self:
```
This is the naming convention used elsewhere, and `llvmPackages` is ambiguous – e.g. using `llvmPackages` in `callPackage` arguments inside the scope would not give this scope and would be misleading. Not a hard blocker, but I think this would be less confusing.